microsoft/terminal
# Add keyboard shortcuts to increase and decrease font size

## Summary of the Pull Request
This hooks up the already existing shortcut actions for increasing and decreasing font size so that keyboard shortcuts can be used. Also adds default shortcuts `ctrl+=` and `ctrl+-` for increasing and decreasing font size.

 
## References
A previous PR was done for this feature but was closed and never completed: #1740 . This is a continuation of that work.


## PR Checklist
* [x] Closes #1708 
* [ ] Tests added/passed. **No new tests added**


## Detailed Description of the Pull Request / Additional comments
There were already actions `increaseFontSize` and `decreaseFontSize` that existed but without anything to handle them. The handler shares existing functionality for adjusting font size that already exists with ctrl and scrolling.


## Validation Steps Performed
1. Add keybindings to `profiles.json` for `decreaseFontSize` and `increaseFontSize`
2. Verified that both keybindings work.
3. Verified that ctrl + scrolling still works.


~~1. Delete `profiles.json` and let terminal generate a default one.~~
~~2. Verified that the default keybindings are created.~~ No longer needed with `defaults.json`
